src/plugins/gc-subway/page2-en.hbs

```diff
@@ -0,0 +1,58 @@
+---
+{
+	"title": "[Page 2] - GC Subway map menu (provisional)",
+	"language": "en",
+	"altLangPrefix": "page2",
+	"pageclass": "cnt-wdth-lmtd",
+	"noContentTitle": true,
+	"breadcrumb": [
+		{ "title": "Canada.ca", "link": "https://www.canada.ca/en.html" }
+	],
+	"secondlevel": false,
+	"dateModified": "2020-12-07",
+	"share": "true"
+}
+---
+{{>alert-provisional-demos}}
+
+<nav class="provisional gc-subway">
+	<h1 id="gc-document-nav">Service initiation template</h1>
```

Comment:
Can we replace **Service initiation template** by **Subway navigation component**?